app/views/notices/shop_employer_notices/initial_employer_reminder_to_publish_plan_year.html.erb

```diff
@@ -0,0 +1,122 @@
+<%= render partial: 'notices/shared/logo' %>
```

Comment:
Changes Required in the following file. It doesn't update the Health Connector Logo.